NDTensors/src/diag/diagtensor.jl

```diff
@@ -33,6 +31,19 @@ function Array(T::DiagTensor{ElT,N}) where {ElT,N}
   return Array{ElT,N}(T)
 end
 
+function DiagonalArrays.diagview(T::NonuniformDiagTensor)
+  return data(T)
```

Comment:
@mtfishman Should these go somewhere else like a `diagview.jl` file?

Reply:
I don't think that is necessary.